NYTimes.com.js

```diff
@@ -90,14 +94,46 @@ function scrape(doc, url) {
 			}
 		}
 		item.url = ZU.xpathText(doc, '//link[@rel="canonical"]/@href') || url;
-		var pdfurl = ZU.xpathText(doc, '//div[@id="articleAccess"]//span[@class="downloadPDF"]/a[contains(@href, "/pdf")]/@href');
+		item.libraryCatalog = "NYTimes.com";
+		// Convert title/tags of NYT archives when in all caps: 
+		if (item.title == item.title.toUpperCase())
+				item.title = ZU.capitalizeTitle(item.title, true);
+		var allcaps = false;
+		for (i=0; i < item.tags.length; i++) {
+			if (item.tags[i] == item.tags[i].toUpperCase())
+				allcaps = true;
```

Comment:
I should have had the cases switches (assume all caps until proven otherwise)—I meant to avoid the acronym situation you mentioned. If I were to wipe `allcaps`/the second loop, wouldn't that lead to that exact situation (title-casing acronyms), as `ZU.capitalizeTitle` wouldn't be able to distinguish acronyms from all-caps tags?